nixos/modules/services/networking/nixops-dns.nix

```diff
@@ -0,0 +1,67 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  pkg = pkgs.nixops-dns;
+  cfg = config.services.nixops-dns;
+in
+
+{
+  options = {
```

Comment:
One more thing that needs to be considered - nixops database location. Current nixops-dns implementation will always look into current user home dir https://github.com/kamilchm/nixops-dns/blob/master/main.go#L23
Do you have an idea how to make it usable as system service?

Reply:
I've tried to address this via user option (The user the nixops-dns daemon should run as.) which propagates to serviceConfig `User = cfg.user;` bellow. Not sure about the group tho.